2017/Team-Training/OMERO-web/index.html

```diff
@@ -0,0 +1,566 @@
+
+
+                <section>
+                    <h2>OMERO.web framework</h2>
+
+                    <img style="height:500px" src="https://www.openmicroscopy.org/site/support/omero5.3/_images/OmeroWeb.png"/>
+                    <br>
+                    <a target="_blank" href="https://www.openmicroscopy.org/site/support/omero5/developers/Web.html">OMERO.web documentation</a>
+                </section>
+
+
+                <section>
+                    <h2>Relationship between apps</h2>
+
+                    <img style="height:500px" src="images/OmeroWeb-core-apps.png"/>
+
+                    <br>
+                    <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/">components/tools/OmeroWeb/omeroweb/</a>
+                    <!-- <ul>
+                        <li>webgateway, webclient, webadmin, feedback, api in
+                            <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/tree/develop/components/tools/OmeroWeb/omeroweb">components/tools/OmeroWeb/omeroweb</a>.
+                        </li>
+                        <li>
+                            Configured in <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/settings.py">OmeroWeb/omeroweb/settings.py</a>
+                        </li>
+                        <li>OMERO.web core apps never installed independently.</li>
+                        <li>Django apps <i>CAN</i> link to or import from other installed apps</li>
+                        <li>
+                            webgateway provides <a target="_blank" href="http://www.openmicroscopy.org/site/support/omero5/developers/Web/WebGateway.html">core functionality</a>: 
+                            <ul>
+                                <li>Class LoginView extended by webclient & api</li>
+                                <li>html templates extended by webclient</li>
+                                <li>views.full_viewer() imported by webclient</li>
+                                <li>webgateway/imgData/id JSON used by webclient, figure, iviewer</li>
+                                <li>webgateway/render_image/id used by webclient, figure, iviewer</li>
+                            </ul>
+                        </li>
+                    </ul> -->
+                </section>
+
+
+                <section>
+                    <h2>OMERO.web for developers</h2>
+
+                    <ul>
+                        <li>clone openmicroscopy/openmicroscopy and build</li>
+                        <li>Run OMERO.web <a target="_blank" href="https://docs.openmicroscopy.org/latest/omero/developers/Web/Deployment.html">as described for developers</a>.
+                        </li>
+                        <li>Use nginx in development? Under discussion: <a target="_blank" href="https://trello.com/c/VdQ4HAl6/65-using-nginx-in-development">See trello card</a>.
```

Comment:
we have used NGINX in our doc

Reply:
https://docs.openmicroscopy.org/latest/omero/developers/Web/Deployment.html describes both with/without nginx